back/api/lib/graphql/collections/users/resolves.js

```diff
@@ -22,24 +22,32 @@ module.exports = {
 
     delete user.password; // don't save plain password!
 
-    return db('put', {
+    let putItem = db('put', {
       TableName: usersTable,
       Item: user
-    })
-    // let's invoke another lambda asynchronously (don't wait till it finished)!
-    .then(() => invoke('timeout', {user, delay: 70}))  // no actual delay here
-    // if we pass a callback it will run synchronously, so we'll get a response
-    .then(() => invoke('timeout', {user, delay: 50}, (response) => {
-      // this should be delayed for 50ms
-      // let's do something with the response
-      if (response.result === 'success') {
-        console.log("response data:", response);
-      } else {
-        return Promise.reject(new Error("Something went wrong :("));
-      }
-    }))
-    // finally return the user record
-    .then(() => user);
+    });
+
+    if(!process.env.IS_OFFLINE) {
+      return putItem
+        // let's invoke another lambda asynchronously (don't wait till it finished)!
+        .then(() => invoke('timeout', {user, delay: 70}))  // no actual delay here
+        // if we pass a callback it will run synchronously, so we'll get a response
+        .then(() => invoke('timeout', {user, delay: 50}, (response) => {
```

Comment:
Yes, that is just an example how to use `invoke`. And because `lambda.invoke`, which 'invoke' uses, tries to run lambda function in AWS instead of local environment, the example invocation is skipped if env variable IS_OFFLINE is set to true. It took me some time to remember what this was all about.